pkg/agent/proxy/proxier.go

```diff
@@ -101,15 +101,16 @@ type proxier struct {
 	// oversizeServiceSet records the Services that have more than 800 Endpoints.
 	oversizeServiceSet sets.String
 
-	runner               *k8sproxy.BoundedFrequencyRunner
-	stopChan             <-chan struct{}
-	ofClient             openflow.Client
-	routeClient          route.Interface
-	nodePortAddresses    []net.IP
-	hostGateWay          string
-	isIPv6               bool
-	proxyAll             bool
-	endpointSliceEnabled bool
+	runner                          *k8sproxy.BoundedFrequencyRunner
+	stopChan                        <-chan struct{}
+	ofClient                        openflow.Client
+	routeClient                     route.Interface
+	nodePortAddresses               []net.IP
+	hostGateWay                     string
+	isIPv6                          bool
+	proxyAll                        bool
+	endpointSliceEnabled            bool
+	svcInternalTrafficPolicyEnabled bool
```

Comment:
I think we should "service" instead of "svc" to match the feature gate name

Reply:
"serviceInternalTrafficPolicyEnabled" is ok to me. I have thought of using "service" instead of "svc", if so, the variable is too long, so I used "svc"